### ISSUE TITLE
Nierówne przeliczniki co do conquest capa w arena bracketach.

### ISSUE THREAD

**Issue by Myrcool** [Atlantiss/BugTracker#6854]
Pressed that Enter by mistake, will edit it asap.

**Comment by aneq** [comment]
wzór na mniejszy cap jest nadal w cliencie gry

**Comment by Myrcool** [comment]
> aneq commented 2 minutes ago

wzór na mniejszy cap jest nadal w cliencie gry

Grasz na serwie, w którego cliencie piszę, że cap na rbgach i arenach jest taki sam więc bym się na to zbytnio nie patrzył.

**Comment by blinkmaster** [comment]
nawet jezeli tak jest, to mam szczera nadzieje, ze tak zostanie. moze to zacheci graczy do dalszego grania 3v3.
2v2 stinks

**Comment by crosik** [comment]
Racja: http://www.arenajunkies.com/topic/179536-conquest-point-cap-vs-personal-rating-chart/

Ale tak jak pisal blink wczesniej, lepiej zeby zostalo tak jak jest moze to zacheci graczy do grania 3v3
